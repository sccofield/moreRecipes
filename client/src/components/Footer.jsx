import React from 'react';

const Footer = () => (
  <footer className="footer">
    <div className="container">
      <div className="row">
        <div className="col">
          <p className="text-center">
            Copyright. All right reserve. More Recipe.
          </p>
        </div>
      </div>

    </div>
  </footer>
);

export default Footer;
