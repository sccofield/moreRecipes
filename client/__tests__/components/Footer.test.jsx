import React from 'react';
import toJSON from 'enzyme-to-json';
import Footer from '../../src/components/Footer';


describe('Footer component', () => {

  it('should render the footer', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper.find('div').length).toEqual(3);
    expect(wrapper.find('p').text()).toBe('Copyright. All right reserve. More Recipe.');
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});

