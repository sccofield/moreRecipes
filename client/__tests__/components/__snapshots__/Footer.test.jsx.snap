// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render the footer 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <p
          className="text-center"
        >
          Copyright. All right reserve. More Recipe.
        </p>
      </div>
    </div>
  </div>
</footer>
`;
